js/google_ima.js

```diff
@@ -2436,7 +2451,7 @@ require("../html5-common/js/utils/utils.js");
      * @method TemplateVideoWrapper#unmute
      */
     this.unmute = function() {
-      _ima.setVolume(volumeWhenMuted);
+      _ima.setVolume(volumeWhenMuted ? volumeWhenMuted : 1);
```

Comment:
Do we need to handle the case where the video is muted while volume is 0?

Reply:
Ok nevermind, looking at the unit tests below it looks like this is deliberate right?

Reply:
Yep, this is deliberate. IMA considers 0 as muted.

Reply:
This covers the case where you mute the player twice without unmuting in between, right?